### DISCUSSION TITLE
How Do I Change My FileBrowser I.P Address?

### DISCUSSION THREAD

**Discussion by TvHead9752** [filebrowser/filebrowser#3403] [Q&A]
Hey! I’ve got FileBrowser running on my PC though NSSM, works fine. I want to be able to change the default I.P in the database to 0.0.0.0. I tried using the default commands (filebrowser.exe config set -a 0.0.0.0) but nothing changes. What could the issue be?

**Comment by maxlivi** [comment]
It is working on linux:

./filebrowser config cat |grep Address

```
2024/08/23 18:46:35 Using database: /home/max/permanent/FILEBROWSER/filebrowser.db
  Address:       0.0.0.0

```
and  also  starting it from command line with  -a .0.0.0.0
